bin/lint-staged

```diff
@@ -34,6 +34,11 @@ cmdline
   .option('-x, --shell', 'Skip parsing of tasks for better shell support')
   .option('-q, --quiet', 'Disable lint-staged’s own console output')
   .option('-d, --debug', 'Enable debug mode')
+  .option(
+    '-p, --concurrent <parallel tasks>',
```

Comment:
I did -p for parallel, since -c was used for config. Thoughts?

Reply:
Would it be ok to call this option `--parallel`, specifying the number of parallel tasks?

Just to re-cap: looking at Listr's source code, the supported falues are `true` for `Infinity`, `false` for `1`, and then any number: https://github.com/SamVerschueren/listr/blob/master/index.js#L39

I'm wondering if since the `beta` branch contains other breaking changes, maybe the default could be changed to `false`/`1` as well.

Reply:
Open to feedback. I named it `concurrent` to match the `listr` parameter, but it is easily changed.

I know there are a bunch of people waiting for this functionality that can't use 9.x. What do you think of releasing this with the default as `true` to maintain existing behaviour with 9.x and then changing the default to `false` for the next breaking release.

Reply:
Yeah that works for me.

Reply:
Thinking about it more, I’d say usually there shouldn’t be any conflicting linters, so the current default is at least just as well-intentioned; might not be worth it to change.